package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Front end for legal representation journey within CMC",
   "private": true,
   "engines": {
-    "node": ">=7.8.0"
+    "node": ">=8.1.4"
```

Comment:
We're using `8.9.0` in Docker, might be worth using that.

Reply:
updated to use 8.9.0